### ISSUE TITLE
Various bugs in Jupyter Interactive (No mouse click, no keyboard typing)

### ISSUE THREAD

**Issue by chang0959** [microsoft/vscode-jupyter#15943]
## Environment data

-   VS Code version: 1.92.2 (Universal)
-   Jupyter Extension version (available under the Extensions sidebar): v2024.7.0
-   Python Extension version (available under the Extensions sidebar): v2024.12.3
-   OS (Windows | Mac | Linux distro) and version: Mac Apple Silicon M1 (Darwin arm64) 
-   Python and/or Anaconda version: No Anaconda, python version : 3.10.9
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): pyenv virtualenv
-   Jupyter server running: Local 

### 1. 
The cursor should be able to come out and drag in the middle of the mouse-clicked part, but when you click, the cursor is focused on the last line of the code. 

### 2.
While working on the Pandas dataframe, if I click sort function on the output value, I can't work because I can't click on the mouse or enter the keyboard in the jupyter cell. 

### 1. 
https://github.com/user-attachments/assets/fc5513c6-c5e6-4ed4-8f9f-6fbefea65e35

### 2. 

https://github.com/user-attachments/assets/515a9571-6dad-4aa4-8d4b-32599cd7092b

## Steps to reproduce:

[**NOTE**: Self-contained, minimal reproducing code samples are **extremely** helpful and will expedite addressing your issue]

1. XXX

## Logs

<details>

<summary>Output for <code>Jupyter</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Jupyter</code>)
</summary>

<p>

```
XXX
```

</p>
</details>

**Comment by amunger** [comment]
>  if I click sort function on the output value

What output renderer are you using to get the sorting feature? (if you click the `...` output menu and select "change presentation", it should show which is currently selected)

**Comment by chang0959** [comment]
I checked and found that it is "text/plain - Data Table (Random Factalslnc.vscode-data-table)". 
By the way, I think it works well with "VSCode Builtin Notebook Output Render (vscode.builtin-notebook-render)". 
Thank you for your confirmation. 

Then, do you know why the first problem [The cursor should be able to come out and drag in the middle of the mouse-clicked part, but when you click, the cursor is focused on the last line of the code.] doesn't work?


![스크린샷 2024-08-21 오전 9 57 01](https://github.com/user-attachments/assets/4a8616cb-a6ff-4489-a654-7e7f51869ccc)

**Comment by amunger** [comment]
yes, the cursor placement in the output is a known issue (though I can't find the issue at the moment)
